#153825434 Redesign Landing Page UI

larrystone opened the pull request:
### What does this PR do?
Redesign landing page with effective UI/UX
### Description of Task to be completed?
- Gather and prepare static assets
- Design and implement landing page template
### How should this be manually tested?
- Launch app via [Reaction commerce](https://elendil-rc-staging.herokuapp.com)
### What are the relevant pivotal tracker stories?
[#153825434](https://www.pivotaltracker.com/story/show/153825434)
### Screenshots (if appropriate)

<img width="1680" alt="screen shot 2018-01-12 at 8 31 45 am" src="https://user-images.githubusercontent.com/8871565/34864160-3669d040-f773-11e7-9f36-143f48e3b800.png">

<img width="1680" alt="screen shot 2018-01-12 at 8 35 19 am" src="https://user-images.githubusercontent.com/8871565/34864235-917944ca-f773-11e7-97b4-459f822d0116.png">


Questions:
N/A

larrystone committed:
feat(landingPage): redesign landing page ui
- gather landing page assets
- create landing page html templates
- override existing landing page (core) with new template
[Delivers #153825434]

larrystone committed:
feat(landingPage): redesign landing page ui
- gather landing page assets
- create landing page html templates
- override existing landing page (core) with new template
[Delivers #153825434]

HamdalahA commented on imports/plugins/included/ui-homepage/client/templates/newArrivals.html:
```diff
@@ -0,0 +1,200 @@
+      <div class="line"></div>
+    </header>
+  </section>
+</template>
```
Good job so far but some of the files needs a newline at the end of the file

larrystone committed:
feat(landingPage): redesign landing page ui
- gather landing page assets
- create landing page html templates
- override existing landing page (core) with new template
[Delivers #153825434]

proxiex reviewed (approved):
good job
